locale/de_DE/submission.xml

```diff
@@ -22,6 +22,8 @@
 	<message key="submission.workflowType.description">Eine Monografie ist ein Werk, das von einer Autorin/einem Autor oder mehreren Autor/innen gemeinsam verfasst worden ist. Ein Sammelband besteht aus mehreren Kapiteln jeweils unterschiedlicher Autor/innen (detaillierte Kapitelangaben sind erst zu einem späteren Zeitpunkt im Veröffentlichungsprozess zu machen).</message>
 	<message key="submission.workflowType.editedVolume">Sammelband</message>
 	<message key="submission.workflowType.authoredWork">Monografie</message>
+	<message key="submission.workflowType.editedVolume.selectEditors">Select Volume Editors</message>
+	<message key="submission.workflowType.editedVolume.selectEditors.description">Select the authors who should be identified as editors for the entire volume.</message>
```

Comment:
This looks like English in the German locale file?

Reply:
@NateWr 
I just filled it, cause I wasn't sure, the correct translation offically.
Should  I delete it, will then Marco automatically detect ?

Reply:
Yeah, typically we only add locale strings for the `en_US` locale and then the translators can see what strings are missing.

When we edit a locale string, if the change is significant, we delete it from other locales so that translators are forced to retranslate it.

Reply:
cool, thanks, will commit in a bit

Reply:
This commit deletes them https://github.com/pkp/omp/pull/687/commits/98ac16a8847c1d4d1bbf2c5f3c4c32c75aadaf92

Reply:
@NateWr   these commits
https://github.com/pkp/omp/pull/687/commits/43f2e36adad93c38b7b6bd6c32f65e56e0c27985
https://github.com/pkp/omp/pull/687/commits/98ac16a8847c1d4d1bbf2c5f3c4c32c75aadaf92

adds the changes.